client/components/formatted-header/style.scss

```diff
@@ -1,4 +1,4 @@
-.step-header {
+.formatted-header {
```

Comment:
Since you're changing this `className`, you should make sure to change it in some other places where it's used. Search for `.step-header` and you'll find a couple of usages in:

* `client/jetpack-connect/style.scss`
* `client/signup/steps/plans/style.scss`